release_images.yml

```diff
@@ -2,182 +2,45 @@
 release_images:
 # Check .release_images_template.yml for reference
   1:
-    framework: "huggingface_pytorch"
-    version: "1.10.2"
+    framework: "tensorflow"
+    version: "2.7.0"
```

Comment:
I think it does not have the `cuda_version` entry in it. Can you confirm if it is required or not?